app/_Game.js

```diff
@@ -99,11 +99,16 @@ class Game {
         }
 
         vm.runInContext("add_log = console.log", game_context);
+
+        vm.runInContext("mode.log_calls=true;mode.log_incoming=true;", game_context);
```

Comment:
`mode.log_calls=true;mode.log_incoming=true; `
shouldn't be used in the master branch as it floods the logs